SingularityUI/app/styles/forms.styl

```diff
@@ -3,4 +3,10 @@
 
 label
 	&.label-light
-		font-weight normal
\ No newline at end of file
+		font-weight normal
+
+.cmd-description
+	margin-top 10px
+
+#cmd-confirm
+	color #5cb85c
```

Comment:
lets try to use variables for colors, and try our best to keep the color pallete to a reasonable number (though if it makes sense to add new ones feel free) - https://github.com/HubSpot/Singularity/blob/master/SingularityUI/app/styles/colors.styl